script/bigscreenplayer.js

```diff
@@ -132,10 +132,11 @@ define('bigscreenplayer/bigscreenplayer',
 
         subtitles = Subtitles(
           playerComponent,
-          bigscreenPlayerData.media.captionsUrl,
+          bigscreenPlayerData.media.captions || { captionsUrl: bigscreenPlayerData.media.captionsUrl },
```

Comment:
This looks like it extends the data model passed in to bigscreen player, maybe worth commenting and/or updating the docs

Reply:
We spoke about this briefly. I think ideally we would converge on a single data model ie. `captions` object. But given that would be a breaking API change, we can guard against it for now, document it and look to simplify in the future. 

Unless we think we can just update this now, do the docs and make it clear in semver the breaking change?

Reply:
Yes, we plan on updating the docs for BSP with live subtitle information. We intentionally built it this way so it worked with the old method and VOD subs wouldn't require testing for our earlier upstream changes. My plan was to create more work to be done after that was go to one data model. I didn't expect us to make so many VOD changes as part of this work but I still feel we made the right decision to break up the work this way.

Reply:
Yep, I agree @Luke-Chatburn. I think this was a sensible approach 🚀